src/repository/storage/postgres/orm/sequelize/noteRelations.ts

```diff
@@ -163,6 +163,24 @@ export default class NoteRelationsSequelizeStorage {
     return affectedRows > 0;
   }
 
+  /**
+   * Deletes parent relation by note id
+   *
+   * @param noteId - id of the current note
+   */
+  public async deleteRelation(noteId: NoteInternalId): Promise<boolean> {
```

Comment:
Maybe "deleteParentRelation" fits better? What if we'll have a "deleteChildRelation" later

Reply:
maybe `unlinkParent()`?